tests-invoke

```diff
@@ -44,6 +44,9 @@ def main():
     def detect_collisions(opts):
         nonlocal last_updated_at
 
+        if opts.pull_number is None:
+            return None
+
```

Comment:
This returning None is akward, I know it's already there. Kinda expected this to be a `-> bool`.